unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs

```diff
@@ -1,4 +1,4 @@
-#if UNITY_IPHONE && !UNITY_5_0_0
+#if !UNITY_5_0_0
 #define SWRVE_USE_HTTPS_DEFAULTS
 #endif
```

Comment:
Hmm, I might turn that into !(UNITY_IPHONE && UNITY_5_0_0) as there is an iOS issue where https just doesn't work on 5.0.0